.scalafmt.conf

```diff
@@ -0,0 +1,85 @@
+version                                  = 3.6.1
```

Comment:
should also `addSbtPlugin("org.scalameta" % "sbt-scalafmt" % "2.5.0")` in plugins.sbt